projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -219,7 +270,7 @@ export class OwcJsonService {
    * (array)   value: '2000-01-01T00:00:00.000Z,2001-01-01T00:00:00.000Z,2002-01-01T00:00:00.000Z,...'
    * (single) value: '2016-01-01T00:00:00.000Z/2018-01-01T00:00:00.000Z/P1Y'
    */
-  getTimeValueFromDimensions(value: string | null) {
+  parseISO8601Values(value: string | null) {
```

Comment:
I think this is normally a breaking Change.
All Methods not marked as private are considered [public as default](https://www.typescriptlang.org/docs/handbook/2/classes.html#public)

Reply:
reverted!